scrapers/lethalpass.yml

```diff
@@ -0,0 +1,29 @@
+name: lethalpass
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lethalpass.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //div[@class="title"]/h1
```

Comment:
from a quick test
title doesnt work for episodes from dvds `https://www.lethalpass.com/5956970` 
HD is returned most of the times.

Reply:
Bugger, was so focused on the exclusive site name I did not notice the Title changing.